nvfuser/.gitignore

```diff
@@ -0,0 +1 @@
+__init__.py
```

Comment:
This is not related, but I feel it is convenient because this is generated every time when I run tests...

Reply:
I have no idea what this file does, but it was removed recently (https://github.com/csarofeen/pytorch/pull/2337/files#diff-e512bea3b02f75ab1e81b759562879c5867e6e863679d6e7696fa34087dc3dc9L1). Is it still generated locally?

Reply:
Yes, it keeps being regenerated.